src/main/kotlin/carracing/RacingMethod.kt

```diff
@@ -0,0 +1,18 @@
+package carracing
+
+import carracing.view.InputView
+import kotlin.random.Random
+
+class RacingMethod {
+    var eachCarList = MutableList(InputView.numberOfCar, { i -> 0 })
+
+    fun raceEachRound(): MutableList<Int> {
+        val numberOfCar = InputView.numberOfCar
+
+        for (i in 0..numberOfCar - 1) {
+            var racingDistance = Random.nextInt(9)
+            if (racingDistance >= 4) eachCarList[i]++
```

Comment:
상수는 Magic Number로 남기면 안된다는 말이 있어
저 9라는 숫자도 나중에 100이나 50으로 바뀐다면 일일이 다 바꿔야 되자나
companion object 내에서 상수 선언해주셔서 바꾸면 좋을 것 같습니다

```suggestion
            var racingDistance = Random.nextInt(ANY_NUMBER)
            if (racingDistance >= NAME_YOU_WANT) eachCarList[i]++
            
            companion object {
                private const val ANY_NAME = 9
                private const val NAME_YOU_WANT = 4
            }
```